src/cljc/matcher_combinators/config.cljc

```diff
@@ -0,0 +1,47 @@
+(ns matcher-combinators.config
+  "Global output behavior configurations"
```

Comment:
figure at some point we could also implement something that reads `~/.config/clojure/matcher-combinators.edn` config file. But for now folks can include `(matcher-combinators/enable-redaction!)` in their `dev/user.clj` startup file to get the same behavor